workflows/h5ad-to-arrow/README.md

```diff
@@ -0,0 +1 @@
+See [containers/h5ad-to-arrow](https://github.com/hubmapconsortium/portal-containers/blob/master/containers/h5ad-to-arrow/README.md).
```

Comment:
I think github would resolve relative links...
```suggestion
See [containers/h5ad-to-arrow](../../containers/h5ad-to-arrow/README.md).
```
... but I'm not sure this actually makes things simpler?

Reply:
It should, but I prefer the full urls since if we were to ever move this file out of Github, the full URL is helpful.